contrib/format-httpd/src/main/resources/bootstrap-format-plugins.json

```diff
@@ -5,7 +5,7 @@
       "formats": {
         "httpd" : {
           "type" : "httpd",
-          "logFormat" : "%h %l %u %t \"%r\" %s %b \"%{Referer}i\" \"%{User-agent}i\"",
+          "logFormat" : "common\ncombined",
```

Comment:
Nice!  I like this a lot better than having to specify the exact config.  Can we put something in the `README` about this?  Are there other canned configs we could use?

Reply:
I already did that. Please check if it is enough documentation (and clear enough for others).